csharp/ql/lib/ext/System.Data.model.yml

```diff
@@ -54,6 +54,17 @@ extensions:
       - ["System.Data", "EnumerableRowCollectionExtensions", False, "Where<TRow>", "(System.Data.EnumerableRowCollection<TRow>,System.Func<TRow,System.Boolean>)", "", "Argument[0].Element", "ReturnValue.Element", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "get_Item", "(System.String)", "", "Argument[this].Element", "ReturnValue", "value", "manual"]
       - ["System.Data", "IColumnMappingCollection", True, "set_Item", "(System.String,System.Object)", "", "Argument[1]", "Argument[this].Element", "value", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetDataTypeName", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetDateTime", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
+      - ["System.Data", "IDataRecord", True, "GetFieldType", "(System.Int32)", "", "Argument[this]", "ReturnValue", "taint", "manual"]
```

Comment:
This API returns a `System.Type`. Do we need this model?

Reply:
No, we probably shouldn't. It appears that java already exclude generating models like this. I will do the same for C#!
Thank you :-)

Reply:
We should probably also introduce neutrals for `GetGuid` and `GetName` to avoid false positives.